PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+1. Describe what this Pull Request does and why you think it is needed.
+   If this PR includes UI or CLI changes, please include Before/After screenshots
+   If this PR includes performance changes, please include Before/After metrics showing improvement.
+-->
+
+<!--
+2. If this fixes an existing issue, please specify in `Fixes #<id>` format
+   (As described in https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
+-->
+
+<!--
+3. Tell @miq-bot to label this PR with an appropriate scope label (bug, enhancement, etc)
```

Comment:
Any reason for this item to be different from the issue templates?

Reply:
In the issue template we know ahead of time if it's a bug or enchantment, since those are the 2 choices we offer.  For PRs we don't know what it's for.